package.json

```
@@ -16,7 +16,8 @@
     "changeset": "packages/cli/bin.js",
     "check-all": "yarn test && yarn lint && yarn types:check",
     "version-packages": "changeset version && yarn prettier",
-    "release": "yarn build && changeset publish"
+    "release": "yarn build && changeset publish",
+    "codesandbox:build": "yarn build && yarn version-packages"
```

I think `version-packages` would fail if there were changesets on CodeSandbox CI since it won't have a GitHub token for the changelog generation. We should just be able to remove it.
```suggestion
    "codesandbox:build": "yarn build"
```

---

Good catch, I've fixed that. The PR is ready for re-review